app/components/explosions/explosions.rb

```diff
@@ -0,0 +1,50 @@
+module Explosions
+  class Explosions < ViewComponent::Base
```

Comment:
this class is great! i love how clearly it sets up the state and manages it through the motion and update

Reply:
Thank you :bowing_woman: